Packs/Malware/pack_metadata.json

```diff
@@ -78,4 +78,4 @@
         "xsoar",
         "marketplacev2"
     ]
-}
\ No newline at end of file
+}
```

Comment:
Why does it show as changed?

Reply:
My `ide` automatically adds `newline` to the end of files (previously there was none). I should remove it, disregard the change.